.github/workflows/release.yaml

```diff
@@ -0,0 +1,50 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Release to NPM
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    tags:
+      - '*'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+      # Set up Node
+    - name: Use Node 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+        registry-url: 'https://registry.npmjs.org'
+
+      # Run install dependencies
+    - name: Install dependencies
+      run: npm install
+
+      # Build extension
+    - name: Run build
+      run: npm run build
+
+      # Run tests
+    - name: Run Test
+      run: npm run test
+
+      # Publish to npm
+    - run: |
+        yarn check-dependencies
+        yarn publish --access public
```

Comment:
What is the difference between what happens here and the on in CI.yaml?
Why not combine them?

Reply:
On CI.yaml the action is run on every commit to push to next whereas here it's only ran when a tag is created and pushed to the repo when

Reply:
CI.yaml reads to me that if the push is successful, then it will create the tag. Why not just add this to that section? 
Maybe I'm missing it.

Reply:
We do not create tag from CI, we create tag manually when we want to do release.
`release.yaml` workflow will run only on that tag and made release on npm registry.